doc/admin/deployment.md

```diff
@@ -1106,16 +1114,131 @@ Replace 16G with the desired tmpfs size.
 
 Agent Configuration
 -------------------
+This section addresses how to configure the DAOS servers on the storage
+nodes before starting it.
+
+### Certificate Generation
+
+The DAOS security framework relies on certificates to authenticate
+administrators. The security infrastructure is currently under
+development and will be delivered in DAOS v1.0. Initial support for certificates has been added to DAOS and can be disable either via the command line or in the DAOS Agent configuration file.
```

Comment:
sp: "can be disabled"

Additionally, just a nit, but the line length should probably be broken up to match up to the others in this paragraph.